pages/security/resources/radical-servers/en.text

```diff
@@ -154,19 +154,6 @@ So far, the following services are provided:
 
 German language, English and French and a little Spanish supported
 
-h2. indivia.net
```

Comment:
indivia.net still exists, and their link works. 

i think if they wish to be removed, the request should come from them.